internal/flybarman/node.go

```diff
@@ -139,8 +139,8 @@ wal_retention_policy = main
 		return fmt.Errorf("failed to write file %s: %s", n.RootPasswordConfigPath, err)
 	}
 
-	barmanCronFileContent := `* * * * * /usr/local/bin/barman_cron
-`
+	barmanCronFileContent := `* * * * * /usr/local/bin/barman_cron`
```

Comment:
I guess we can delete the cron stuff?

Reply:
The current plan is to not touch any of the flybarman stuff within this PR.  We will have to come up with a sane deprecation strategy that doesn't catch anyone using it too off guard.